pyqir-generator/pyqir/generator/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
```

Comment:
For my edification, what's the role of the empty init file here? Is that a common pattern in python development?

Reply:
This marks the `pyqir/generator` folder as a package; otherwise, the folder is effectively invisible to Python.

Regardless, though, it might make sense to re-export items from submodules of `pyqir.generator` here so that `import pyqir.generator` is useful without necessarily needing to remember which parts are in `module` or in `qis`.

Reply:
[Every Python package must contain an `__init__.py` file.](https://docs.python.org/3/tutorial/modules.html#packages) If there's nothing to do to initialize the package, it can be empty.

Reply:
> Regardless, though, it might make sense to re-export items from submodules of `pyqir.generator` here so that `import pyqir.generator` is useful without necessarily needing to remember which parts are in `module` or in `qis`.

@cgranade I think that if we made everything available in `pyqir.generator` module, I would want to hide/remove the submodules, so that there's only one way to import them. Does that make sense?

Reply:
Created #56 to follow up on this.